packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts

```diff
@@ -429,22 +429,20 @@ describe('NegotiatorHelpers', () => {
         ])
         .instance();
 
-      const actualOperationConfig = helpers.negotiateOptionsForDefaultCode(httpOperation, desiredOptions)(logger);
+      const actualOperationConfig = helpers.negotiateOptionsForUnspecifiedCode(httpOperation, desiredOptions)(logger);
 
       expect(helpers.negotiateOptionsBySpecificResponse).toHaveBeenCalledTimes(1);
       assertRight(actualOperationConfig, operationConfig => {
         expect(operationConfig).toEqual(fakeOperationConfig);
       });
     });
 
-    it('given no 2xx response should throw exception', () => {
+    it('given no 2xx response should return the first response', () => {
       const desiredOptions = { dynamic: false };
       jest.spyOn(helpers, 'negotiateOptionsBySpecificResponse');
 
-      const negotiationResult = helpers.negotiateOptionsForDefaultCode(httpOperation, desiredOptions)(logger);
-      assertLeft(negotiationResult, e => {
-        expect(e.name).toBe('https://stoplight.io/prism/errors#NO_SUCCESS_RESPONSE_DEFINED');
-      });
+      const negotiationResult = helpers.negotiateOptionsForUnspecifiedCode(httpOperation, desiredOptions)(logger);
+      assertRight(negotiationResult, res => expect(res.code).toBe('300'));
```

Comment:
This emphasizes how severe this change is. Are we still sure that it is not a major change?
// @philsturgeon